app/controllers/providers/proceedings_sca/proceeding_issue_statuses_controller.rb

```diff
@@ -0,0 +1,46 @@
+module Providers
+  module ProceedingsSCA
+    class ProceedingIssueStatusesController < ProviderBaseController
+      prefix_step_with :proceedings_sca
+
+      def show
+        @proceeding = legal_aid_application.proceedings.last
+        form
+      end
+
+      def update
+        return continue_or_draft if draft_selected?
+
+        @proceeding = legal_aid_application.proceedings.last
+
+        if form.valid?
+          return redirect_to providers_legal_aid_application_sca_interrupt_path(legal_aid_application, "proceeding_issue_status") unless form.proceeding_issue_status?
```

Comment:
This will fall through to the `default` handling in the view as there is no specific set of `proceeding_issue_status` entries under interrupts.  Was this by design?

Reply:
It was not. I messed up a rebase at one point and think i forgot to redo this. I'll fix it. Good spot!